tests/acceptance/request_test.py

```diff
@@ -79,6 +84,13 @@ def test_echo_date_with_json_renderer(test_app):
         assert response.json == input_object
 
 
+@pytest.mark.parametrize('body', [NoDefault, {}])
+def test_post_endpoint_with_optional_body(test_app, body):
+    expected_length = len(dumps(body)) if body is not NoDefault else 0
+    # This is brutal for now ... it will be used to at first for debugging the issue
```

Comment:
I don't get this comment.

Reply:
... testing leftover :)
The test was really brutal at first ... if the call does not raise error I'm happy :)